front/utils/utils.ts

```diff
@@ -68,8 +68,11 @@ export function formatNumber(value: number, options?: Intl.NumberFormatOptions):
     maximumFractionDigits: 2,
     ...options,
   } as const;
+  
+  return formatFrench(value, defaultOptions)};
 
-  return formatFrench(value, defaultOptions);
+export function firstLetterUppercase(str: string) {
```

Comment:
```suggestion
export function formatFirstLetterToUppercase(str: string) {
```

Une fonction commence plutot par un verbe par convention en Functional Programming

Reply:
Oui très juste, merci.
Done